src/stan/mcmc/hmc/hamiltonians/base_hamiltonian.hpp

```diff
@@ -37,29 +37,50 @@ namespace stan {
       }
 
       // The time derivative of the virial, G = \sum_{d = 1}^{D} q^{d} p_{d}.
-      virtual double dG_dt(Point& z) = 0;
+      virtual double dG_dt(
+        Point& z,
```

Comment:
I'd strongly prefer arguments start on the same line as the function when they'll fit to reduce vertical space.